rts/Lua/LuaUtils.cpp

```diff
@@ -980,6 +980,10 @@ static bool ParseCommandOptions(
 		return true;
 	}
 
+	if (lua_isnoneornil(L, idx)) {
+		return true;
```

Comment:
Perhaps best be explicit?
```suggestion
	if (lua_isnoneornil(L, idx)) {
		cmd.SetOpts(0);
		return true;
```

Reply:
Thought so, but since it's 0 by default don't see the need.

Reply:
Can we rely on the default? I guess in case somebody passes an already-modified object it's good to keep whatever value it already has anyway.

Reply:
I'll check just in case but I think we're creating all just in time.